src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -30,6 +30,7 @@
         "symfony/stopwatch": "~2.3|~3.0.0",
         "symfony/templating": "~2.1|~3.0.0",
         "symfony/translation": "~2.8",
+        "doctrine/cache": "~1.0",
```

Comment:
I'm pretty sure this is a mistake. We only reference hard-dependencies and Doctrine cache is only used AFAICS for the validator and that's even optional.